e2e/cypress/tests/support/ui/account_settings_modal.d.ts

```diff
@@ -27,7 +27,7 @@ declare namespace Cypress {
          *       // Do something here
          *   });
          */
-        uiOpenProfileModal(): Chainable;
+        uiOpenProfileModal(section: string = ''): Chainable;
```

Comment:
```suggestion
        uiOpenProfileModal(section?: string): Chainable;
```
Can you also update return type instead of `any`
maybe `uiOpenProfileModal(section: string = ''): Chainable<Jquery>;`
or 
`uiOpenProfileModal(section: string = ''): Chainable<Element>;`
depends on definition

Reply:
is the return type `Chainable<void>`?
<img width="805" alt="image" src="https://user-images.githubusercontent.com/7455516/197378522-6ddf75da-9a67-4d0b-aca8-889be5062654.png">